tempo_project/dimensions/supplemental/states.csv

```diff
@@ -0,0 +1,50 @@
+id,name,is_conus,census_division,census_region
+AL,Alabama,True,east_south_central,south
+AR,Arkansas,True,west_south_central,south
+AZ,Arizona,True,mountain,west
+CA,California,True,pacific,west
+CO,Colorado,True,mountain,west
+CT,Connecticut,True,new_england,northeast
+DC,District of Columbia,True,south_atlantic,south
```

Comment:
I assume you have this sorted out elsewhere, but just noting DC = state here, and AK HI PR are not listed